AudioRecorder.py

```diff
@@ -1,6 +1,11 @@
 import custom_speech_recognition as sr
-import pyaudiowpatch as pyaudio
 from datetime import datetime
+import os
+
+if os.name == 'nt':
```

Comment:
1) Try to refactor to set up all differences between windows and MAC in one place. Dependency injection looks appropriate here open for other approach's to keep code clean.